incubator/gitea/templates/deployment.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
```

Comment:
In Kubernetes 1.10 this API version can be dropped. Since this Deployments have gone to `apps/v1beta1`, `apps/v1beta2`, and most recently `apps/v1`. In 1.10 anything prior to `apps/v1beta2` can be dropped. This is the version I would recommend specifying unless you want to support multiple versions by checking in Capabilities.APIVersions for a version.

Reply:
See other comment, sorry! I'm following the other stable charts as well for motivation. Seems to be in-line with how the others do it.

Reply:
To add some more context, there is a [deprecation policy page](https://kubernetes.io/docs/reference/deprecation-policy/) with details and examples. Based on the timing, this API has not been removed due to a bug. 

A bunch of the charts were created with `extensions/v1beta1`. One of the TODOs we need to tackle for the charts repo is updating them before the API version goes away. To have less work when that happens we've been asking charts to use a newer API version.